src/lib/replication/repl_dev/raft_repl_dev.h

```diff
@@ -22,14 +23,17 @@ struct raft_repl_dev_superblk : public repl_dev_superblk {
     uint32_t raft_sb_version{RAFT_REPL_DEV_SB_VERSION};
     logstore_id_t free_blks_journal_id; // Logstore id for storing free blkid records
     uint8_t is_timeline_consistent; // Flag to indicate whether the recovery of followers need to be timeline consistent
-    uint64_t last_applied_dsn;      // Last applied data sequence Number
+    uint64_t last_applied_dsn;      // Last applied data sequence number
+    uint8_t destroy_pending;        // Flag to indicate whether the group is in destroy pending state
```

Comment:
I didn't see any place init this field. Shouldn't we initialize it to false, or is the whole superblk structure already zeroed somewhere before passed to RaftReplDev constructor? A specific init won't might be better here.

Reply:
It is initialized as part of the metablk created time, in RaftReplDev::RaftReplDev() and they are written whenever the cp is flushed.

Reply:
In RaftReplDev constructor it is a std::move, right? https://github.com/eBay/HomeStore/blob/master/src/lib/replication/repl_dev/raft_repl_dev.cpp#L33, I don't see any zero in the superblk class `create` api, have I missed anything?